parlai/crowdsourcing/tasks/model_chat/conf/example.yaml

```diff
@@ -12,7 +12,7 @@ mephisto:
     include_persona: false
     conversation_start_mode: 'hi'
     annotation_question: Does this comment from your partner have any of the following attributes? (Check all that apply)
-    conversations_needed_string: "blender_90M:10"
+    conversations_needed_string: "blender_3B:2"
```

Comment:
let's keep the example the same - we want the casual user of model chat to not have to load the large Blender model